src/Umbraco.Web.UI.Client/src/views/components/umb-groups-builder.html

```diff
@@ -81,7 +81,7 @@
                         <i class="icon icon-merge"></i>
                         <localize key="contentTypeEditor_inheritedFrom"></localize>: {{ tab.inheritedFromName }}
                         <span ng-repeat="contentTypeName in tab.parentTabContentTypeNames">
-                            <a href="#/settings/documentTypes/edit/{{tab.parentTabContentTypes[$index]}}">{{ contentTypeName }}</a>
+                            <button class="btn-link btn-small p0" ng-click="openDocumentType(tab.parentTabContentTypes[$index])">{{ contentTypeName }}</button>
```

Comment:
Make sure to give this one a `type="button"` otherwise it will submit when this is inside of a `<form>` element.

Reply:
I've updated it for ya! :-)